content/nginxaas-azure/faq.md

```diff
@@ -39,7 +39,7 @@ Common questions about F5 NGINX as a Service for Azure (NGINXaaS).
 ### I am an NGINX Plus customer; how can I switch to NGINXaaS?
 - In NGINX Plus, customers SSH into the NGINX Plus system, store their certificates in some kind of storage and configure the network and subnet to connect to NGINX Plus.
 
-- For NGINXaaS, customers store their certificates in the Azure key vault and configure NGINXaaS in the same VNet or peer to the VNet in which NGINXaaS is deployed.
+- For NGINXaaS, customers store their certificates in the Azure key vault and configure NGINXaaS in the same VNet or peer to the VNet in which NGINXaaS is deployed.  There is a Portal option to upload your existing configuration as a single archive.
```

Comment:
Good call out. I believe this option (to upload existing NGINX configuration along with aux files, certificates, etc as an archive) is also available through other clients and is not specific just to the portal.